src/System.Net.Security/src/System/Net/Unix/CertModule.cs

```diff
@@ -132,10 +132,33 @@ internal override X509Certificate2 GetRemoteCertificate(SafeDeleteContext securi
         //
         internal override string[] GetRequestCertificateAuthorities(SafeDeleteContext securityContext)
         {
-            // TODO (Issue #3362) populate issuers
-            string[] issuers = Array.Empty<string>();          
+            using (SafeSharedX509NameStackHandle names = Interop.libssl.SSL_get_client_CA_list(securityContext.SslContext))
+            {
+                if (names.IsInvalid)
+                {
+                    return Array.Empty<string>();
+                }
+
+                int nameCount = Interop.Crypto.GetX509NameStackFieldCount(names);
+
+                if (nameCount == 0)
+                {
+                    return Array.Empty<string>();
+                }
 
-            return issuers;
+                string[] clientAuthorityNames = new string[nameCount];
+
+                for (int i = 0; i < nameCount; i++)
+                {
+                    using (SafeSharedX509NameHandle nameHandle = Interop.Crypto.GetX509NameStackField(names, i))
+                    {
+                        X500DistinguishedName dn = Interop.Crypto.LoadX500Name(nameHandle);
```

Comment:
Looking at [the referenced code](https://github.com/dotnet/corefx/blob/master/src/System.Net.Security/src/System/Net/_SecureChannel.cs#L511-L524), yes.  [The code for get_Issuer](https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslX509CertificateReader.cs#L54-L57) is the same: `LoadX500Name(nameHandle).Name`